src/strings/en-us.json

```diff
@@ -24,6 +24,8 @@
     "AllowMediaConversionHelp": "Grant or deny access to the convert media feature.",
     "AllowOnTheFlySubtitleExtraction": "Allow subtitle extraction on the fly",
     "AllowOnTheFlySubtitleExtractionHelp": "Embedded subtitles can be extracted from videos and delivered to clients in plain text in order to help prevent video transcoding. On some systems this can take a long time and cause video playback to stall during the extraction process. Disable this to have embedded subtitles burned in with video transcoding when they are not natively supported by the client device.",
+    "AllowFfmpegThrottling": "Allow slowing down transcoding when it gets ahead enough",
+    "AllowFfmpegThrottlingHelp": "When transcoding (or remuxing) gets ahead enough from current playback position, slow down the process so it would consume less resources. Useful when watching without much seeking around. Try turning it off if you experience playback issues.",
```

Comment:
```suggestion
    "AllowFfmpegThrottling": "Throttle Transcodes",
    "AllowFfmpegThrottlingHelp": "When transcodes or remuxes gets far enough ahead from the current playback position, pause the process so it will consume less resources. This is most useful when watching without seeking often. Turn this off if you experience playback issues.",
```

Reply:
multiline suggestions!